script.js

```diff
@@ -0,0 +1,63 @@
+const nav = document.getElementById('top-navigation-container');
+const closeIcon = document.getElementById('close-icon');
+const hamburger = document.getElementById('hamburger');
+const portfolio = document.getElementById('portfolioRef');
+const about = document.getElementById('aboutRef');
+const contact = document.getElementById('contactRef');
+
+hamburger.addEventListener('click', () => {
+  closeIcon.classList.remove('hide');
+  nav.classList.add('nav');
+  about.classList.add('big-font');
+  contact.classList.add('big-font');
+  portfolio.classList.add('big-font');
+  closeIcon.classList.add('close-icon');
+});
+
+closeIcon.addEventListener('click', () => {
+  closeIcon.classList.add('hide');
+  about.classList.add('top-navigation');
+  contact.classList.add('top-navigation');
+  portfolio.classList.add('top-navigation');
+  nav.classList.remove('nav');
+  about.classList.remove('big-font');
+  contact.classList.remove('big-font');
+  portfolio.classList.remove('big-font');
+  closeIcon.classList.remove('close-icon');
+});
+
+portfolio.addEventListener('click', () => {
+  closeIcon.classList.add('hide');
+  about.classList.add('top-navigation');
+  contact.classList.add('top-navigation');
+  portfolio.classList.add('top-navigation');
+  nav.classList.remove('nav');
+  about.classList.remove('big-font');
+  contact.classList.remove('big-font');
+  portfolio.classList.remove('big-font');
+  closeIcon.classList.remove('close-icon');
+});
+
+about.addEventListener('click', () => {
+  closeIcon.classList.add('hide');
+  about.classList.add('top-navigation');
+  contact.classList.add('top-navigation');
+  portfolio.classList.add('top-navigation');
+  nav.classList.remove('nav');
+  about.classList.remove('big-font');
+  contact.classList.remove('big-font');
+  portfolio.classList.remove('big-font');
+  closeIcon.classList.remove('close-icon');
+});
+
+contact.addEventListener('click', () => {
+  closeIcon.classList.add('hide');
+  about.classList.add('top-navigation');
+  contact.classList.add('top-navigation');
+  portfolio.classList.add('top-navigation');
+  nav.classList.remove('nav');
+  about.classList.remove('big-font');
+  contact.classList.remove('big-font');
+  portfolio.classList.remove('big-font');
+  closeIcon.classList.remove('close-icon');
+});
```

Comment:
- [ ] Great job implementing the functionality of the `Menu` 👌💯, but I suggest following the [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)
you've repeated the same lines of code in every function and It's not the best practice, To keep your code clean and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) I suggest combining all the same lines of code in one function and using it 👌😉

> For Example 

```
const openMenu = () => {
  closeIcon.classList.remove('hide');
  nav.classList.add('nav');
  about.classList.add('big-font');
  contact.classList.add('big-font');
  portfolio.classList.add('big-font');
  closeIcon.classList.add('close-icon');
};

const closeMenu = () => {
  closeIcon.classList.add('hide');
  about.classList.add('top-navigation');
  contact.classList.add('top-navigation');
  portfolio.classList.add('top-navigation');
  nav.classList.remove('nav');
  about.classList.remove('big-font');
  contact.classList.remove('big-font');
  portfolio.classList.remove('big-font');
  closeIcon.classList.remove('close-icon');
};

hamburger.addEventListener('click', openMenu);

closeIcon.addEventListener('click', closeMenu);
portfolio.addEventListener('click', closeMenu);
about.addEventListener('click', closeMenu);
contact.addEventListener('click', closeMenu);
```